package.json

```
@@ -16,6 +16,7 @@
     "lint": "npm-run-all --continue-on-error \"lint:*\"",
     "lint:dependency-version-consistency": "check-dependency-version-consistency .",
     "lint:eslint": "eslint --report-unused-disable-directives \"**/*\"",
+    "lint:lockfile-lint": "lockfile-lint --path yarn.lock --allowed-hosts yarn --allowed-schemes \"https:\"",
```

I have banned usage of `git+ssh:` links in `package.json` via `--allowed-schemes \"https:\"`. If we need to install something from GitHub directly, we’ll need to do this:

```suggestion
    "lint:lockfile-lint": "lockfile-lint --path yarn.lock --allowed-hosts yarn --allowed-schemes \"https:\" \"git+ssh:\"",
```

I have also banned `registry.npmjs.com` which should not see anyway if we use `yarn add` etc. If this becomes a problem, we can do:

```suggestion
    "lint:lockfile-lint": "lockfile-lint --path yarn.lock --allowed-hosts npm yarn --allowed-schemes \"https:\"",
```

Both constraints speed up `yarn install` and leave us with fewer _single points of failure_.